descriptors/ModuleDescriptor-template.json

```diff
@@ -706,7 +706,7 @@
     },
     {
       "id": "finance.transactions",
-      "version": "1.0 2.1"
+      "version": "1.0 2.0 3.0"
```

Comment:
I've realized that orders must support only finance.transactions version 3

Reply:
changed to 3.0 only